SlicerConfiguration/Settings/ActiveSliceSettings.cs

```diff
@@ -52,7 +52,7 @@ public class ActiveSliceSettings
 		public static RootedObjectEventHandler ActiveProfileModified = new RootedObjectEventHandler();
 		public static RootedObjectEventHandler SettingChanged = new RootedObjectEventHandler();
 
-		private static PrinterSettings activeInstance = PrinterSettings.Empty;
```

Comment:
I think defining the Empty profile was done on purpose as described in: 
https://github.com/MatterHackers/MatterControl/commit/a566e745f286f573b6a60ee54beb824dcca87c70